AlgebraInLean/Chapter01/Sheet04.lean

```diff
@@ -80,12 +82,13 @@ protected def C₃.op : C₃ → C₃ → C₃
 | .rot0, .rot120 => .rot120
 | .rot0, .rot240 => .rot240
 | .rot120, .rot0 => .rot120
-| .rot120, .rot120 => .rot240
+| .rot120, .rot120 => .rot240 -- rotating by 120° and again by 120° is equivalent to a 240° rotation
 | .rot120, .rot240 => .rot0
 | .rot240, .rot0 => .rot240
 | .rot240, .rot120 => .rot0
 | .rot240, .rot240 => .rot120
```

Comment:
Following from your previous comment, it might be nice to state something similar here, bringing attention to the fact that 480° = 120° + 360°.

Reply:
I don't understand which line this would refer to

Reply:
The line `| .rot240, .rot240 => .rot120`.

Reply:
done